front/app/containers/CustomPageShow/CustomPageProjectsAndEvents.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import EventsWidget from 'components/LandingPages/citizen/EventsWidget';
+import { isNilOrError } from 'utils/helperUtils';
+import useAdminPublications from 'hooks/useAdminPublications';
+import { ICustomPageData } from 'services/customPages';
+import ContentContainer from 'components/ContentContainer';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+import { PublicationStatus } from 'services/projects';
+import useAdminPublicationsStatusCounts from 'hooks/useAdminPublicationsStatusCounts';
+import ProjectAndFolderCardsInner from 'components/ProjectAndFolderCards/ProjectAndFolderCardsInner';
+
+interface Props {
+  page: ICustomPageData;
+}
+
+const CustomPageProjectsAndEvents = ({ page }: Props) => {
+  // There will be either topic or area ids if this component renders.
+  // To enable it, the page needs either a topic or area associated with it.
+  const topicIds = page.relationships.topics.data.map((topic) => topic.id);
+  const areaIds = page.relationships.areas.data.map((area) => area.id);
+  const publicationStatusFilter: PublicationStatus[] = [
+    'published',
+    'archived',
+  ];
+
+  const adminPublications = useAdminPublications({
+    topicIds,
+    areaIds,
+    publicationStatusFilter: publicationStatusFilter,
+  });
+
+  const paginatedAdminPublications = useAdminPublications({
```

Comment:
Unfortunately, it seems we should fix it. Probably, we should not fetch all publications on this page.

Reply:
What should be fixed? We could have a CustomPageProjects and CustomPageEvents component that handles the data loading for each.

Reply:
> What should be fixed?

We fetch admin publication twice. 

> We could have a CustomPageProjects and CustomPageEvents component

The data will still be loaded twice.

But maybe it's not super important.

Reply:
Yeah, except for when events or projects are off, while the other one is off. Then it'll only request for the component that is turned on.

What are you suggesting we do?